test/e2e/radio/radio.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { StoryBookPage } from '../storybookPage';
+import { Page, test, expect } from '@playwright/test';
+
+const getPage = (page: Page) =>
+  new StoryBookPage(page, '?path=/story/next-radio--interactive');
+
+test.describe('Radio - Standard Size', () => {
```

Comment:
Can we have a flat structure here? The reason is easier to search the string whenever we have a failed test.

Reply:
sure like this: https://github.com/glints-dev/glints-aries/blob/v4/test/e2e/alert/alert.spec.ts

we actually missed that for few components and addressing them here: https://github.com/glints-dev/glints-aries/pull/735

Reply:
Fixed